lib/generators/devise_security/install_generator.rb

```diff
@@ -4,7 +4,7 @@ module DeviseSecurity
   module Generators
     # Generator for Rails to create or append to a Devise initializer.
     class InstallGenerator < Rails::Generators::Base
-      LOCALES = %w[ en de it ]
+      LOCALES = %w[ en es de it tr ].freeze
```

Comment:
please add `fr` to this list before the `it` and the tests will pass.  Also please remove extra spaces at start and end.

Reply:
👍 done